src/app/planning/trip-create/trip-create.component.html

```diff
@@ -1 +1,2 @@
 <h1>{{ 'headings.create-trip' | translate }}</h1>
+<app-trip-form></app-trip-form>
```

Comment:
any reason why you decided to separate TripCreateComponent from TripFormComponent?